test/module/shared_model/validators/field_validator_test.cpp

```diff
@@ -784,3 +784,45 @@ TEST_F(FieldValidatorTest, QueryContainerFieldsValidation) {
       },
       [] {});
 }
+
+/**
+ * @given field validator with custom config
+ * @when try to give a description with a length of default limit
+ * @then description is valid
+ */
+TEST_F(FieldValidatorTest, TryReachDefaultLimit)
+{
+  validation::FieldValidator custom_field_validator(iroha::test::kTestsValidatorsCustomConfig);
+
+  shared_model::validation::ReasonsGroupType reason;
+  custom_field_validator.validateDescription(reason, std::string(65, 0));
```

Comment:
```suggestion
  custom_field_validator.validateDescription(reason, std::string(kMaxDescriptionSizeDefault, 0));
```
no magic outside Hogwarts!

Reply:
Because test checks that custom value is used in the validation

Reply:
then it should be the custom max size?
```suggestion
  custom_field_validator.validateDescription(reason, std::string(SettingsStub{}.getMaxDescriptionSize(), 0));
```
or (better) some comnstant

Reply:
I think a more correct way of checking that your custom limit works is issuing 2 commands: one with the maximum value and one with maximum + smallest step (1 in our case)

Reply:
fixed